paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentSelection.kt

```diff
@@ -17,15 +17,24 @@ import kotlinx.parcelize.Parcelize
 internal sealed class PaymentSelection : Parcelable {
 
     abstract val requiresConfirmation: Boolean
-    abstract fun mandateText(context: Context): String?
+
+    abstract fun mandateText(
+        context: Context,
+        merchantName: String,
```

Comment:
I didn't see any tests that validated the use of `merchantName`. Did I miss something? Or can we add a test for it?